db/seeds/questionnaires/student_post_assessment.rb

```diff
@@ -1,259 +1,130 @@
-                       id: :v36,
-                       type: :radio,
-                       title: 'Zou je het erg vinden als jouw begeleider op de hoogte wordt gesteld als jij twee of meer metingen mist?',
-                       options: ['Ja', 'Nee'],
-                       section_end: true
-                     }, {
-                       section_start: 'Beloning',
-                       id: :v37,
-                       type: :radio,
-                       title: 'Vind je dat je genoeg beloning krijgt voor wat je moet doen?',
-                       options: ['Ja'],
-                       otherwise_label: 'Nee, voor het werk dat ik heb gedaan zou ik dit een eerlijke beloning vinden: €'
-                     }, {
-                       id: :v38,
-                       type: :radio,
-                       title: 'Je hebt nu drie weken meegedaan aan dit onderzoek. Denk je dat je ditzelfde onderzoek ook voor zeven maanden zou volhouden voor €70?',
-                       options: ['Ja'],
-                       otherwise_label: 'Nee, omdat:',
-                       section_end: true
-                     }, {
-                       section_start: 'Tot slot',
-                       id: :v39,
-                       type: :textarea,
-                       title: 'Wat zou jij willen verbeteren aan de webapp die je de afgelopen drie weken hebt gebruikt?',
-                       section_end: true
-                     }]
+  id: :v1,
+  type: :radio,
+  title: 'Ben je definitief gestopt met je opleding (zonder diploma)?',
+  options: [
+    { title: 'Ja', shows_questions: %i[v2 v3 v4] },
+    'Nee'
+  ],
+  show_otherwise: false
+}, {
+  id: :v2,
+  hidden: true,
+  type: :date,
+  max: [2018, 11, 01],
+  required: true,
+  title: 'Wanneer ben je ongeveer gestopt? Als je het niet precies meer weet, vul dan iets in dat zo goed mogelijk in de buurt komt'
+}, {
+  id: :v3,
+  type: :radio,
+  hidden: true,
+  title: 'Hoeveel jaar moest je nog ongeveer tot je diploma?',
+  options: ['0 tot 1 jaar',
+            '1 tot 2 jaar',
+            '2 tot 3 jaar',
+            '3 jaar of meer'],
+  show_otherwise: true
+}, {
+  id: :v4,
+  type: :radio,
+  hidden: true,
+  title: 'Ben je van plan om met een nieuwe opleiding te starten?',
+  options: [
+    'Ja',
+    'Nee',
+    'Weet ik nog niet'
+  ],
+  show_otherwise: false
+}, {
+  id: :v5,
+  hidden: true,
+  type: :range,
+  title: 'Hoe zeker ben je ervan dat je jouw opleiding gaat afmaken?',
+  labels: ['Helemaal niet', 'helemaal wel']
+}, {
+  id: :v6,
+  type: :checkbox,
+  show_otherwise: false,
+  title: 'Wat voor cijfer sta/stond je gemiddeld voor al je vakken samen?',
+  tooltip: 'Een grove schatting is prima, het hoeft niet precies te kloppen.',
+  options: [
+    { title: 'Weet ik echt niet', hides_questions: %i[v6_1], shows_questions: %i[v6_2] }
+  ]
+}, {
+  id: :v6_1,
+  hidden: false,
+  type: :range,
+  min: 1,
+  max: 10,
+  step: 0.5,
+  title: '',
+  labels: %w[1 10]
```

Comment:
Volgens mij maakt het geen sense dat je 0.5 steps hebt als je alleen de labels 1 en 10 hebt. Waarom niet 1 t/m 10?

Reply:
1 tm 10 werd heel lelijk, maar dat zouden we nog kunnen aanpassen